docs/v13/documentation/branching-journeys-checkboxes.md

```diff
@@ -0,0 +1,147 @@
+  text: "Continue"
+}) }}
+
+</form>
+```
+
+Some things to note:
+
+The form `action` is "/exports-answer", and the `method` is "post", so this will make a 'post' request to the path "/exports-answer" - we will need a route to respond to that request.
+
+In the checkboxes component, the `name` is "exports". The name is important - it is where the kit stores the answer to the question. Names can't have spaces so use hyphens instead, and use lowercase for simplicity.
+
+The button needs to be between the 2 `form` tags. Do not add an `href` to the button - this will turn it into a link and means it no longer submits the form.
+
+The page looks like this:
+
+![What do you export? Select all that apply. Checboxes: Fruit Vegetables Meat. Button: Continue](/public/docs/v13/images/docs/branching-checkboxes.png)
+
+## The route
+
+We will look at 2 example routes for checkboxes:
+
+1. Check if an answer is selected (regardless of other answers)
+2. Check if a specific answer is selected (and no others)
+
+### Example 1: A route to check if an answer is selected
+
+In `app/routes.js` we need to add a route to process the answer from the form.
+
+```
+router.post('/exports-answer', function(request, response) {
+
+	var exports = request.session.data['exports']
+	if (exports.includes("Fruit")){
+		response.redirect("/fruit")
+	} else {
+		response.redirect("/next-question")
+	}
+})
+```
+
+`router.post`
+
+This route is for a post request - the `method` we set on the form.
+
+`'/exports-answer'`
+
+This route is for a request to '/exports-answer' - the 'action' we set on the form.
+
+`var exports = request.session.data['exports']`
+
+The kit stores all the data from forms in `request.session.data`, using the 'name' we used in the component, in this case `exports`. We copy it to a shorter variable (country) so it's easier to type later on.
+
+`if (exports.includes("Fruit")){`
+
+Checkboxes are always stored as lists (called 'arrays') in the Prototype Kit. This code checks whether the array stored for exports includes "Fruit" - the 'value' on one of our checkboxes. Note that this will also be true if any of the other checkboxes were ticked. We'll see how to check for only specific answers in a later example.
+
+`response.redirect("/fruit")`
+
+Send a redirect as a response to the browser - sending it to the path "/fruit" - a page with specific questions about fruit.
+
+`} else {`
+
+If 'exports' does not include 'Fruit', run the code after:
+
+`response.redirect("/next-question")`
+
+Send a redirect as a response to the browser - sending it to the path "/next-question" (or whatever path your next question is).
+
+### Example 2: A route to check if only a specific answer is selected
+
+Add this line at the top of `routes.js`:
+
+```
+const util = require('util')
+```
+
+Then add this route:
+
+```
+router.post('/exports-answer', function(request, response) {
+
+	var exports = request.session.data['exports']
+	if (util.deepStrictEqual(exports, ['Fruit'])){
```

Comment:
@nataliecarey we need this to compare arrays, but I'd like to make it nicer.

I'm wondering if we make `utils.deepStrictEqual` available under our own path and naming. Something like:
```
const arraysAreEqual = govukPrototypeKit.utils.arraysAreEqual
```
then
```
if (arraysAreEqual(exports, ['Fruit']))
```